webapp/core/ImportProject.js

```diff
@@ -47,6 +47,55 @@ var ImportProject = function(json){
       transaction: t
     };
     
+    var nameAlreadyInUse = function(objectName, type){
+      var restriction = { name: objectName };
+      switch(type){
+        case 'provider':
+          var providers = DataManager.listDataProviders(restriction);
+          if (providers.length > 0)
+            return Promise.resolve(true);
```

Comment:
This expression can be simplified. 

`return Promise.resolve(providers.length > 0);`